lara-typescript/src/example-interactive/src/components/runtime.tsx

```diff
@@ -31,6 +32,27 @@ export const RuntimeComponent: React.FC<Props> = ({initMessage}) => {
     setCustomMessages(messages => [...messages, msg]);
   }, { "*": true });
 
+  useDecorateContent((msg: any) => {
+    const domClasses = ["text-decorate"];
+    const options = {
+      words: msg.words,
+      replace: msg.replace,
+    };
+
+    const decoratedContentClickListener = {
+      type: "click",
+      listener: (evt: Event) => {
+        const wordElement = evt.srcElement as HTMLElement;
+        if (!wordElement) {
+          return;
+        }
+        const clickedWord = (wordElement.textContent || "").toLowerCase();
+        decoratedContentEvent({type: "click", text: clickedWord});
```

Comment:
```suggestion
        decoratedContentEvent({type: "click", text: clickedWord, bounds: wordElement.getBoundingClientRect()});
```
Based on my suggestion on the glossary-plugin PR, here's where you could pass the bounds of the element.

Reply:
I like it.  Added.